content/changes.md

```diff
@@ -8,6 +8,50 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
+## January 31, 2024 
+
+### Features 
+ 
+- **Qualified and Learnosity Integration** 
+
+   We've partnered with Learnosity to integrate Qualified coding challenges into Learnosity assessments! Our custom question type enables students to solve project and classic coding challenges in a Learnosity assessment, powered by Qualified's Embed engine. 
+
+  Check our post [Qualified and Learnosity Partnership: Bridging the Skills Gap with Developer Assessments](https://andela.com/blog-posts/qualified-and-learnosity-partnership-bridging-the-skills-gap-with-developer-assessments) to learn more.
+
+### Improvements 
+
+- **Erlang 26** 
+
+    Erlang 26 is now supported within project challenges. 
+
+- **Elixir 1.15** 
+
+    Elixir 1.15 is now supported within project challenges.
+
+- **Add deprecated language version labels** 
+
+    A new "deprecated" label was added to old language versions in the challenge edit and detail views. This helps content managers identify which language versions will be removed in the future. For further information and tips on upgrading your content, please check our [guide for upgrading language versions](https://docs.qualified.io/creating-content/challenges/upgrading-language-versions/#upgrading-language-versions).
+  
+### Bug Fixes
+
+- **Fixed report card "Portfolio" tab performance issue**
+
+  :::note
+  Signals customers only
+  ::: 
+
+  Users were experiencing heavy lag, especially with scrolls, once the Portfolio tab was activated and the number of assessments results was 6 or more. This issue was successfully fixed.  
```

Comment:
Please be careful with the leading whitespace here and check the deploy preview (or preview locally):

![image](https://github.com/andela-technology/qualified-docs/assets/154474514/efede544-f2ed-4f58-8596-ca9378c9837c)

```suggestion
    Users were experiencing heavy lag, especially with scrolls, once the Portfolio tab was activated and the number of assessments results was 6 or more. This issue was fixed.
```

Really minor, but also try to avoid trailing spaces. These are usually removed by pre commit autoformatting, but we don't use it on our markdown files since we have non-standard extensions.